src/interpolation.js

```diff
@@ -143,25 +148,29 @@ export function range (color1, color2, options = {}) {
 
 	outputSpace = outputSpace? ColorSpace.get(outputSpace) : space;
 
-	color1 = color1.to(space).toGamut();
-	color2 = color2.to(space).toGamut();
+	color1 = to(color1, space);
+	color2 = to(color2, space);
+
+	// Gamut map to avoid areas of flat color
```

Comment:
Should document that GM happens before interpolation here, especially as this is a change from CSS Color 4